### PR TITLE
Fixed Ubuntu Installation Dependency

### PR DESCRIPTION
**Pull request by DEVLOPRR**
`docs/contributing/framework-developer-guide.mdx`
- Replaced Old Ubuntu Dependency `libappindicator3-dev` with `libayatana-appindicator3-dev` Since The Old Dep is Been [Removed](https://tracker.debian.org/pkg/libappindicator).
- Removed `armhf` and `arm64` Since The Build Script Only Accepts `ia32` and `x64`.
- Removed Extra Spaces in Command.
- Removed `'`.

`docs/getting-started/your-first-neutralinojs-app.mdx`
- Added Information About what `--release` argument does in `neu build` command.
- Removed Info Since Application.
- INFO: 

`src/pages/index.js`
- Replaced Platform Support "Chrome" with "Chrome Browser" since people were [confusing](https://youtu.be/H5XQ7CsQc-Y?t=128) it with Chrome OS.
- Replaced Text With The Appropriate URL.

### CODE REVIEW

File: docs/getting-started/your-first-neutralinojs-app.mdx [hunk 1]
```diff
@@ -142,10 +142,11 @@ Congrats! your application works.
 
 ![mypp is running on Linux](../media/myapp-running-linux.png)
 
-:::info
+<!-- Currently Commented Since This Statement is false, as application doesn't reload, i am trying to figure out why. -->
```
Comment (shalithasuranga) [inline docs/getting-started/your-first-neutralinojs-app.mdx:145]: There was a small issue with the minimal template. I fixed it with https://github.com/neutralinojs/neutralinojs-minimal/commit/752e3ab094e357f61331d76be61b3ad4d85f88b9

Could you please check by creating a new app, and make this note visible again, so we can merge this pull request ;)

**Review by shalithasuranga — COMMENTED** [review]
Thank you very much for this pull request. Could you please check and resolve the inline comment

**Comment by DEVLOPRR** [comment]
it's fixed now!

**Comment by shalithasuranga** [comment]
Hey.. @DEVLOPRR  Could you please check the above inline comment. Seems like the info box is still commented out ;) Thanks

**Comment by shalithasuranga** [comment]
Hey.. @DEVLOPRR  Could you please check the above inline comment. Seems like the info box is still commented out ;) Thanks

**Review by DEVLOPRR — COMMENTED** [review]
I've Updated, And Removed The Info Box Comment.

**Review by shalithasuranga — APPROVED** [review]
Thank you very much :+1:

**Pull request merged**